libs/cart/src/drivers/magento/models/inputs/cart-item-with-sku.ts

```diff
@@ -0,0 +1,6 @@
+import { DaffCartItemInput } from '../../../../models/cart-item-input';
+import { DaffCartItem } from '../../../../models/cart-item';
+
+export interface MagentoDaffCartItemInputWithSku extends DaffCartItemInput {
```

Comment:
This should be able to be removed with the other changes mentioned

Reply:
Changed in [`931e356`](https://github.com/graycoreio/daffodil/pull/676/commits/931e356340d5188dea7fc0865152377868fef06b).